front/pages/api/stripe/webhook.ts

```diff
@@ -390,7 +397,7 @@ async function handler(
           if (validStatus.isErr()) {
             logger.error(
               {
-                panic: true,
+                stripeError: true,
```

Comment:
replaced panic by stripeError => one monitor popping, most specific, is the best